packages/radix-vue/src/DismissableLayer/utils.ts

```diff
@@ -170,9 +174,7 @@ export function useFocusOutside(
       }
     }
 
-    ownerDocument.addEventListener('focusin', handleFocus)
-
-    cleanupFn(() => ownerDocument.removeEventListener('focusin', handleFocus))
+    ownerDocumentFocusinCleanup = useEventListener(ownerDocument, 'focusin', handleFocus)
   })
```

Comment:
`useEventListener` inside of `watchEffect` requires cleanup as well

https://play.vuejs.org/#eNqdVduO2zYQ/ZVZv9hBtJLd5Mm1F9ukBpqiaBdpgT5UfdBKI4kbilR5sV0Y+vcMSUkrGckiyAJriDxzOXM4Q14WP7VtfLS42C52OlesNaDR2PYuFaxppTJwAYUldFAq2cCSTJcj9F42bb8fJ27hIi1/TEUqcim0gUZXsHf+q+UvyLmEv6Xixc3y1WBgZFVx7G2MskjILglEiAItDDYtzwzSCmBXb+4uFx+263YJrfwuE601cLxtZIF8ny4ITxeQ+AAEP1pjpID7nLP8E8Fj0pvwlS7uwscuCaYh6pCZArNydCNjhxLuq3dJ6DuZ0JwtFtEiiHXbZG38pKUgpS/OJ+0BnS624Hfc3j0paDUmuVSUaks7tTGt3iZJlZF728RMJqJttlPD+8063mzidcJEgee4eaKY0XVEXWcKi2+KGUxfjEqWtwU27MV4g9E9BXobb9YJZ4/TaC5Yl4qORDKaGqJk1ZVEOWnMOKo/WsOoYWZSZdRQp1/9nuuckVpeY/7pC/tP+hzYPijUqI6k74iZTFVoAnz483c80/cIUltZd/AvgB9RS24dx2D2zoqCaE/sPNsP/sSZqP7Sh7NBoYeiHFGvhrf38rr2+lrpz3TfxG8nKj4qW8dGk37j8NJ5Ho4ozG9MU0JU4yRPG2iZqvm8R3DKTF4fyhJzMx9+snT/YX7/O7k5+mcTwQ8RvPl3APHsQ5VW5I48OF6rV8SfMIB+9BWrKlQPkgmD6md5Eg9Sf6Srpr8wMl3ojG6KiUsurTA9vvZIQOnG+uCCHDO+ojT7uyGTcySXmACLr1+HzS6CzXo99Z/UulpJ8Z5jJmx7HUjQEWPMZeW5LaNp7J4mACthVcjcNiQ53Oz3QI2AJRNYjOUPZl8TIET0zsJy3vs9+3bDp/+Y8jPAh1OmxFcHP9KKYOlvQqpghfMi52U+V9VNxAIYFboWG8b8BCQJBWu8DqZmGiRd0ZkoSGxGnSEVZFDiic6OUhYR/C8t5JmgNYLkxXMl2jDOh26h0ZlzmnOjnNSRMUU/ZHk9p0cYfIco3u3LunhooEGLKZPvPWB/pv6Hxnl4Uq8fZ+CZqNyLRLfC9KF2YzYOa5y4ZXiNw/y9/LQW7Ni/bO+U7f/q8LgFaP6udZ8B0jm9Sw==

Reply:
Yes I forget about other reactive dependencies in watchEffects